Client/Frontend/Browser/TabPeekViewController.swift

```diff
@@ -111,22 +127,6 @@ class TabPeekViewController: UIViewController, WKNavigationDelegate {
         screenShot?.accessibilityLabel = previewAccessibilityLabel
     }
 
-    fileprivate func setupWebView(_ webView: WKWebView?) {
```

Comment:
curious why this was moved into `viewDidLoad`. Separate methods usually help with readability.